views/build.html

```diff
@@ -7,63 +7,70 @@
     <dd>{{resource.metadata.namespace}}</dd>
     <dt>Created</dt>
     <dd>{{resource.metadata.creationTimestamp | date:'medium'}}</dd>
-  </dl>  
+  </dl>
   <h3>Build Configuration</h3>
   <dl class="dl-horizontal" style="margin-bottom: 0;">
     <dt>Strategy</dt>
-    <dd>{{resource.parameters.strategy.type}}</dd>
-  </dl>    
-  <div ng-switch="resource.parameters.strategy.type">
-    <div ng-switch-when="STI">
-      <div ng-if="resource.parameters.strategy.stiStrategy.image">
-        <dl class="dl-horizontal" style="margin-bottom: 0;">
-          <dt>Builder image</dt>
-          <dd>{{resource.parameters.strategy.stiStrategy.image}}</dd>
-        </dl>
-      </div>
-      <div ng-if="resource.parameters.strategy.stiStrategy.from">
+    <dd>{{resource.spec.strategy.type}}</dd>
+  </dl>
+  <div ng-switch="resource.spec.strategy.type">
+    <div ng-switch-when="Source">
+      <div ng-if="resource.spec.strategy.sourceStrategy.from">
         <dl class="dl-horizontal" style="margin-bottom: 0;">
-          <dt>Builder image repo [tag]</dt>
-          <dd>{{resource.parameters.strategy.stiStrategy.from | imageRepoReference : resource.parameters.strategy.stiStrategy.tag}}
-          </dd>                  
+          <dt>Kind</dt>
+          <dd>{{resource.spec.strategy.sourceStrategy.from.kind}}</dd>
+          <dt>Name</dt>
+          <dd>{{resource.spec.strategy.sourceStrategy.from | imageObjectRef : resource.metadata.namespace}}</dd>
+          <dt ng-if="resource.spec.strategy.sourceStrategy.from.namespace">Name</dt>
```

Comment:
Actually, if you're going to use `imageObjectRef` above, we probably don't need a separate line for kind and namespace